### PR TITLE
feat(environment) change the way that the environment script works. N…

### PR DESCRIPTION
**Pull request by kaiomagalhaes**
…ow will be possible to run this script everytime you want to setup the docker containers even when they already exists if so it will only start them

### CODE REVIEW

File: best_practices/environments.md [hunk 1]
```diff
@@ -4,26 +4,80 @@ This is a shell script which basically any engineer can run in development or pr
...
+    attach_to_container
+  else
+    echo 'Creating project container'
+
+    cp Dockerfile.development Dockerfile
+    docker build -t codelittinc/$PROJECT_NAME .
+    rm Dockerfile
+    docker run \
+      -ti \
+      --name outnix-ruby \
```
Comment (halissonvit) [inline best_practices/environments.md]: it should be $PROJECT_CONTAINER_NAME

File: best_practices/environments.md [hunk 2]
```diff
@@ -4,26 +4,80 @@ This is a shell script which basically any engineer can run in development or pr
...
+      -e <main-name-uppercase>_DATABASE_PASSWORD=postgres \
```
Comment (halissonvit) [inline best_practices/environments.md]: upcase the project_container_name
```
$(echo $PROJECT_CONTAINER_NAME | tr 'a-z' 'A-Z')
```

**Comment by kaiomagalhaes** [comment]
@halissonvit @codelitt Can one of you guys take a look?

**Comment by halissonvit** [comment]
good job @kaiomagalhaes

**Comment by codelitt** [comment]
@kaiomagalhaes @halissonvit looks really good. Sorry I didn't see this sooner.

**Comment by kaiomagalhaes** [comment]
@halissonvit thanks!

**Pull request merged**